src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalExternalUserFunctionalTest.java

```diff
@@ -400,10 +426,20 @@ public void findUserWithAccessTypeShouldBeSuccess() {
         log.info("findUserWithAccessTypeShouldBeSuccess :: STARTED");
         Map<String, Object> response = professionalApiClient.searchOrganisationUsersByUserIdExternal(OK,
                 professionalApiClient.getMultipleAuthHeaders(pumBearerToken), activeUserId);
-        validateAccessTypesInRetrievedUser(response, "ACTIVE", true);
+        validateAccessTypesInRetrievedUser(response, "ACTIVE",
+                true,true, activeUserId);
         log.info("findUserWithAccessTypeShouldBeSuccess :: END");
     }
 
+    public void findUserWithIdShouldBeSuccess(Boolean hasAccessType) {
+        log.info("findUserWithIdShouldBeSuccess :: STARTED");
```

Comment:
for this log to e useful, change the text to be different to the previous method?